block/manager.go

```diff
@@ -172,7 +190,7 @@ func (m *Manager) GetFraudProofOutChan() chan *abci.FraudProof {
 }
 
 // AggregationLoop is responsible for aggregating transactions into rollup-blocks.
-func (m *Manager) AggregationLoop(ctx context.Context) {
+func (m *Manager) AggregationLoop(ctx context.Context, lazy bool) {
```

Comment:
[non-blocking][comment] We should consider a clean abstraction/config for a Sequencer Interface. Rollkit could have multiple settings itself and different shared sequencer technologies might emerge as well.

Reply:
Sequencer and fork-choice is the same problem. We now have a fork-choice interface PR